src/components/Quote.js

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+
+const Quotes = () => {
+  const [data, setData] = useState(null);
+  const [error, setError] = useState(null);
```

Comment:
Thanks @BenMukebo I have removed un used codes, But I think the loading state is there, refresh your browser and it will display 

![Screenshot_2023-04-29_02_09_12](https://user-images.githubusercontent.com/100070005/235268078-ee758065-cbfb-468c-bd26-b93d266fae11.png)

Reply:
I'm talking about the state not the brower. Lokk at this example:

![image](https://user-images.githubusercontent.com/79058364/235268756-29ff32d6-652b-46c4-a6da-6e87de97cb5a.png)

Reply:
thanks @BenMukebo, but I think you need to go through all the code in that section for you to understand the functionality I used